app/services/foreman_salt/report_importer.rb

```diff
@@ -72,6 +72,8 @@ def import_log_messages
 
         message = if result['changes']['diff']
                     result['changes']['diff']
+                  elsif !result['pchanges'].blank? && result['pchanges']['diff']
```

Comment:
`!result['pchanges'].blank?` already implies that `result['pchanges']['diff']` is truthy, no need for the second condition